lib/board.rb

```diff
@@ -3,15 +3,11 @@ class Board
     attr_reader :moves
 
     def initialize
-        @moves = Array.new(9)
-    end
-    
-    def empty?
-        moves.all? { |move| move.nil? }
+        @moves = Array.new(9) {|i| i + 1} 
```

Comment:
(1..9).to_a might be cleaner?

Reply:
Ah, thanks! I was trying to incorporate (1..9) but couldn't figure it out.